src/Ting/Driver/Mysqli/Driver.php

```diff
@@ -120,6 +120,7 @@ public function __construct($connection = null, $driver = null)
     {
         if ($connection === null) {
             $this->connection = \mysqli_init();
+            $this->connection->options(MYSQLI_OPT_INT_AND_FLOAT_NATIVE, 1);
```

Comment:
Is this necessary for supporting Geometry type ?

Reply:
No, it is not. It's an extra optimization.

Reply:
Alright ! :+1: